mulighetsrommet-arena-adapter/src/main/kotlin/no/nav/mulighetsrommet/arena/adapter/routes/ApiRoutes.kt

```diff
@@ -17,7 +18,23 @@ fun Route.apiRoutes() {
         val request = call.receive<ReplayTopicEventsRequest>()
 
         JobRunners.executeBackgroundJob {
-            arenaEventService.replayEvents(table = request.table)
+            arenaEventService.replayEvents(
```

Comment:
En ting som også kunne vært interessant å sjekke ut (synes jeg i hvert fall :smile:) er om db-scheduler sin [one-time task](https://github.com/kagkarlsson/db-scheduler#one-time-task) hadde vært en bedre måte å kjøre denne jobben enn via `JobRunners.executeBackgroundJob`.
Dette er jobb som kan ta veldig lang tid, og da kan kanskje db-scheduler hjelpe oss med å sørge for at vi kun kjører én `replayEvents`-jobb av gangen. Dette er kun et internverktøy for vår egen del, så risikoen for at vi gjør dette er i utgangspunktet lav, men kan være litt nyttig læring med å sette noe slikt også :)